packages/gatsby-plugin-gatsby-cloud/package.json

```diff
@@ -25,7 +25,8 @@
     "cross-env": "^7.0.3",
     "jest": "^26.6.3",
     "msw": "^0.29.0",
-    "node-fetch": "^2.6.1"
+    "node-fetch": "^2.6.1",
+    "shx": "^0.3.3"
```

Comment:
can we move to [cpy-cli](https://www.npm.im/cpy-cli) it's more on par with other libs.